enumerables.rb

```diff
@@ -137,6 +149,8 @@ def my_map()
 
   # my_inject
   def my_inject(*args)
+    raise('LocalJumpError.new NO BLOCK OR ARGUMENT GIVEN!') if !block_given? && arguments.empty?
```

Comment:
You are using args and to raise an error you are using 'arguments'. I think this cause the problem for the my_inject.